packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -153,26 +159,24 @@ const renderHeaderContent = ({
 
   const content = (
     <HeadingContentWrapper>
-      {!isOnDemand && (
+      {isLive && (
+        <LiveLabel
+          service={service}
+          dir={dir}
+          liveText={liveLabel}
```

Comment:
Technically, we could use the default prop for `liveText` when is in English.

Reply:
I think I'd rather just pass it in every time - it makes it a bit less complex at this level!